probabilistic.go

```diff
@@ -0,0 +1,1142 @@
+	BFExists(ctx context.Context, key, element interface{}) *BoolCmd
+	BFInfo(ctx context.Context, key string) *BFInfoCmd
+	BFInfoArg(ctx context.Context, key string, option BFInfoArgs) *BFInfoCmd
+	BFInsert(ctx context.Context, key string, options *BFInsertOptions, elements ...interface{}) *BoolSliceCmd
+	BFMAdd(ctx context.Context, key string, elements ...interface{}) *BoolSliceCmd
+	BFMExists(ctx context.Context, key string, elements ...interface{}) *BoolSliceCmd
+	BFReserve(ctx context.Context, key string, errorRate float64, capacity int64) *StatusCmd
+	BFReserveExpansion(ctx context.Context, key string, errorRate float64, capacity, expansion int64) *StatusCmd
+	BFReserveNonScaling(ctx context.Context, key string, errorRate float64, capacity int64) *StatusCmd
+	BFReserveArgs(ctx context.Context, key string, options *BFReserveOptions) *StatusCmd
+	//TODO LoadChunk and ScanDump missing
+
+	CFAdd(ctx context.Context, key, element interface{}) *BoolCmd
+	CFAddNX(ctx context.Context, key, element interface{}) *BoolCmd
+	CFCount(ctx context.Context, key, element interface{}) *IntCmd
+	CFDel(ctx context.Context, key string, element interface{}) *BoolCmd
+	CFExists(ctx context.Context, key, element interface{}) *BoolCmd
+	CFInfo(ctx context.Context, key string) *CFInfoCmd
+	CFInsert(ctx context.Context, key string, options *CFInsertOptions, elements ...interface{}) *BoolSliceCmd
+	CFInsertNx(ctx context.Context, key string, options *CFInsertOptions, elements ...interface{}) *IntSliceCmd
+	CFMExists(ctx context.Context, key string, elements ...interface{}) *BoolSliceCmd
+	CFReserve(ctx context.Context, key string, capacity int64) *StatusCmd
+	//TODO LoadChunk and ScanDump missing
+
+	CMSIncrBy(ctx context.Context, key string, elements ...interface{}) *IntSliceCmd
+	CMSInfo(ctx context.Context, key string) *CMSInfoCmd
+	CMSInitByDim(ctx context.Context, key string, width, height int64) *StatusCmd
+	CMSInitByProb(ctx context.Context, key string, errorRate, probability float64) *StatusCmd
+	CMSMerge(ctx context.Context, destKey string, sourceKeys ...string) *StatusCmd
+	CMSMergeWithWeight(ctx context.Context, destKey string, sourceKeys map[string]int) *StatusCmd
+	CMSQuery(ctx context.Context, key string, elements ...interface{}) *IntSliceCmd
+
+	TOPKAdd(ctx context.Context, key string, elements ...interface{}) *StringSliceCmd
+	TOPKCount(ctx context.Context, key string, elements ...interface{}) *IntSliceCmd
+	TOPKIncrBy(ctx context.Context, key string, elements ...interface{}) *StringSliceCmd
+	TOPKInfo(ctx context.Context, key string) *TOPKInfoCmd
+	TOPKList(ctx context.Context, key string) *StringSliceCmd
+	TOPKListWithCount(ctx context.Context, key string) *MapStringIntCmd
+	TOPKQuery(ctx context.Context, key string, elements ...interface{}) *BoolSliceCmd
+	TOPKReserve(ctx context.Context, key string, k int) *StatusCmd
+	TOPKReserveWithOptions(ctx context.Context, key string, k int, width, depth int64, decay float64) *StatusCmd
```

Comment:
`TopKAdd`, `TopKCount` etc?

Reply:
Yeah, that actually makes sense since "Top" is not an abbreviation. Good point. Will change it.